ChangeLog

```diff
@@ -488,6 +488,9 @@ New deprecations
      deprecated.
    * PSA_ALG_AEAD_WITH_DEFAULT_TAG_LENGTH and PSA_ALG_AEAD_WITH_TAG_LENGTH
      have been renamed, and the old names deprecated.
+   * Signed magnitude and ones' complement implementations for signed integers are
```

Comment:
This is not a deprecation. We have a category `Requirement changes` for this kind of things.

Reply:
This is not addressed.